test/spec/test-node-specific.js

```diff
@@ -186,6 +186,13 @@ describe("tus", function () {
       await assertUrlStorage(storage);
     });
   });
+
+  describe("#InsecureNodeHttpStack", function () {
```

Comment:
Speaking of tests, do you know if it's possible to a test whether the options are properly passed to the request object?

Reply:
unfortunately no. most of the properties added to nodejs's `Request` are stored in private varibales